src/test-build-only/levels.js

```diff
@@ -63,9 +63,9 @@ module.exports = Object.assign({
       "redstoneWire", "redstoneWire", "redstoneWire", "", "", "rails", "rails", "rails", "", "",
       "redstoneWire", "", "redstoneWire", "", "", "", "", "rails", "", "",
       "redstoneWire", "redstoneWire", "redstoneWire", "", "", "", "", "rails", "rails", "",
-      "", "railsRedstoneTorch", "", "", "", "", "rails", "", "rails", "",
+      "", "railsRedstoneTorch", "", "", "pistonDownSticky", "", "rails", "", "rails", "",
       "", "", "", "", "", "", "rails", "rails", "rails", "",
-      "", "", "", "", "", "", "", "", "", "",
+      "", "", "", "", "railsRedstoneTorch", "", "", "", "", "",
```

Comment:
Is this missing the actual unit test?

Reply:
oh bollocks... did I include a change to the level file? 😭

Reply:
Bollocks... did I commit changes to levels? 😭